create_manifest.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+"""
+/*******************************************************************************
+*   Ledger SDK
+*   (c) 2022 Ledger
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+/*******************************************************************************
+"""
+
+import sys
+import argparse
+import tomli_w
```

Comment:
This is not a standard lib I guess? I mean not included by default like sys and argparse?

Reply:
Nop it is not a standard lib 
I made a PR in [ledger_app_builder](https://github.com/LedgerHQ/ledger-app-builder/pull/45) to have it

Reply:
Ok! But as this is not needed by many of external developers (that might not use ledger-app-builder) and our tools (that don't use ledger-app-builder) I suggest calling it only for a specific target, so that it doesn't create trouble when not necessary. I know I told you the opposite last time, sorry.